README.md

```diff
@@ -48,6 +48,9 @@ Clone the github repository:
 
 Start up a selenium server. By default, the tests expect the selenium server to be running at `http://localhost:4444/wd/hub`.
 
+Download [ChromeDriver](https://sites.google.com/a/chromium.org/chromedriver/home) and place it under `./selenium`
```

Comment:
I think this is helpful to note, but it is probably easier to use the `webdriver-manager` executable instead of manually downloading chromedriver. `./bin/webdriver-manager update`